examples/rpc/partner-plugin/docs/example-messages/accommodation_product_info.json

```diff
@@ -0,0 +1,25 @@
+{
+    "header": {
+        "base_header": {
+            "end_user_wallet_address": "esse laboris Lorem nulla consequat",
+            "version": {
+                "major": -700654324,
```

Comment:
The examples should contain meaningful values or leave them out if they're not required. A huge negative major version doesn't make sense.

Same goes for the other version parts, wallet address and so on.

Reply:
Example messages now have more meaningful values.